omise/classes/omise_charge_class.php

```diff
@@ -62,7 +62,7 @@ public function createOffsite($offsiteType)
             'description' => $this->getChargeDescription(),
             'metadata' => $this->getMetadata(),
             'source' => is_array($offsiteType) ? $offsiteType : array('type' => $offsiteType),
-            'return_uri' => $this->getReturnUri(),
+            'return_uri' => $returnUri
```

Comment:
$returnUri is undefined.

Reply:
Yes it is, it's a parameter to the function. There are cases where it is not needed

Reply:
In case of, offsite payments there should be some value, right?
right now it looks like it will be always NULL for all offsite payments.

Reply:
Already fixed in another later commit d173bfb6a94c3a879f8d8e7be8e1a7fba5e98790